h5p-editor-bildetema-words-topic-image/src/components/editor/Editor.tsx

```diff
@@ -351,72 +359,94 @@ export const Editor: FC<EditorProps> = ({ image, words, initialHotspots }) => {
     <div className={styles.editor} data-test-id="editor">
       <span className={styles.editor_label}>{editorLabel}</span>
       <span className={styles.editor_description}>{editorDescription}</span>
+
       <div className={styles.toolbar}>
-        {selectedWordId
-          ? `${selectedWordLabel}: ${getSelectedWordLabel(selectedWordId)}`
-          : selectWordLabel}
-        {selectedWordId && (
-          <div className={styles.toolbar_buttons}>
-            {colors.map(color => (
-              <ColorButton
-                key={color}
-                color={color}
-                handleClick={setHotspotColor}
-                selected={selectedHotspot?.color === color}
-              />
-            ))}
-            {copy && (
-              <button
-                className={styles.button}
-                type="button"
-                onClick={handleCopy}
-              >
-                Copy
-              </button>
-            )}
-            {paste && (
-              <button
-                className={styles.button}
-                type="button"
-                onClick={handlePaste}
-              >
-                Paste
-              </button>
-            )}
-            <button
-              className={styles.button}
-              type="button"
-              onClick={handleFinishedPressed}
-            >
-              {finishedButtonLabel}
-            </button>
-            <button
-              className={styles.button}
-              type="button"
-              onClick={handleReset}
-            >
-              {resetButtonLabel}
-            </button>
-            <button
-              className={styles.button}
-              type="button"
-              onClick={moveHotspotUp}
-            >
-              <ArrowIcon
-                transform="scale(0.9) rotate(180)"
-                transformOrigin="50% 50%"
-              />
-            </button>
-            <button
-              className={styles.button}
-              type="button"
-              onClick={moveHotspotDown}
-            >
-              <ArrowIcon transform="scale(0.9)" transformOrigin="50% 50%" />
-            </button>
-          </div>
-        )}
+        <div>
+          {selectedWordId
+            ? `${selectedWordLabel}: ${getSelectedWordLabel(selectedWordId)}`
+            : selectWordLabel}
+        </div>
+        <div>
+          {selectedWordId && (
+            <div className={styles.wrapper}>
```

Comment:
Since we now have so many buttons in the toolbar, we should make them wrap under each other when the screen width gets smaller, so that they are not cut off
<img width="679" alt="Skjermbilde 2022-11-15 kl  14 36 03" src="https://user-images.githubusercontent.com/35261194/201932975-a1ee16c2-b3cb-4f81-813f-0eae620c290b.png">